src/components/Article/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { PrismicRichText } from "@prismicio/react";
+import { H1, P, MarginContainer } from "../../styles/styles";
+import { ArticleContainer, Section, StyledP, StyledImage } from "./styled";
+
+const Article = ({
+  title,
+  content,
+  author,
+  imageUrl,
+  imageAlt,
+  imageCaption,
+}) => {
+  return (
+    <MarginContainer>
+      <ArticleContainer>
+        <Section>
+          <H1>{title}</H1>
+          <P>{author}</P>
+        </Section>
+        <Section>
+          <StyledImage src={imageUrl} alt={imageAlt} />
+          <P>{imageCaption} </P>
+        </Section>
+        <Section>
+          <PrismicRichText
+            field={content}
+            components={{
+              paragraph: ({ children }) => <StyledP>{children}</StyledP>,
```

Comment:
can you add `heading2` and `heading3` here too? I think it's safe to assume some articles might have headings throughout